apps/browser-extension-wallet/src/hooks/useWalletManager.ts

```diff
@@ -528,6 +574,78 @@ export const useWalletManager = (): UseWalletManager => {
     [getCurrentChainId]
   );
 
+  const createSharedWallet = async ({
```

Comment:
This function doesn't seem to support HW wallets. For those you need to export own xpub by using HW key agents.
```suggestion
  const createInMemorySharedWallet = async ({
```

Reply:
Fixed on aaf64f5